### DISCUSSION TITLE
Trying to run this on my local network

### DISCUSSION THREAD

**Discussion by Explorographer** [lllyasviel/Fooocus#513] [Q&A]
I have a main box that I am using as a server. I want all the other PC/MACs in the house to be able to access Fooocus.  I have the port changed and figured out how to do that through editing the webui.py but cannot figure out how to put the server in --listen mode.  It keeps telling me that --Listen is not defined.  If I change the default value to something different, it tells me that is not defined.  Halp?

**Comment by Explorographer** [comment]
Also, I do not want to list the server on Gradio.

**Comment by phineas-pta** [comment]
avoid edit source code like that

u can just launch from console something like `python entry_with_update.py --listen --port ████`

**Reply by Explorographer** [reply]
Thanks @phineas-pta I ended up figuring out how to do it finally.  Was unaware that you could pass commands in the .bat file.  Weeeee learning!

Oooh, do you know.  Does this auto-update (git pull)?   EDIT: I see that it does...nice!
